Azure/sonic-buildimage
# [201811][platform/cel]: Port fancontrol changes for dx010/e1031 to 201811 branch

**- What I did**
-  Update fancontrol service for Seastone-DX010/E1031 device to support hysteresis temperature threshold and difference config for each unit fan direction type (B2F/F2B) follow master branch

**- How I did it**
- Add fancontrol config file fancontrol-B2F,fancontrol-F2B
- Add fancontrol script fancontrol.sh
- Modify fancontrol daemon to support i2c base config


**- How to verify it**
- Verify fancontrol service status
  - [dx010-201811-fsc-test.log](https://github.com/Azure/sonic-buildimage/files/4856362/dx010-201811-fsc-test.log)
  - [e1031-201811-fsc-test.log](https://github.com/Azure/sonic-buildimage/files/4856368/e1031-201811-fsc-test.log)

- Verify fan speed under thermal chamber 

Signed-off-by: Wirut Getbamrung wgetbumr@celestica.com
